docs/source/contents/4_draft_mapping.rst

```diff
@@ -1777,10 +1777,644 @@ Decision
+
+.. code-block:: xml
+
+    <subject>
+        <geographic>Becky Cable House (Tenn.)</geographic>
+        <cartographics>
+            <coordinates>35.58546, -83.84444</coordinates>
+        </cartographics>
+    </subject>
+
+.. code-block:: turtle
+
+    @prefix dcterms: <http://purl.org/dc/terms/> .
+
+    <https://example.org/objects/1> dcterms:spatial <https://sws.geonames.org/4630912> ;
+        dcterms:spatial "35.58546, -83.84444" .
+
+Name values with roleTerms represented as subjects
+--------------------------------------------------
+
+Use Case
+^^^^^^^^
+
+The Arrowmont Simple Images collection includes mods:subject/mods:name/mods:namePart values with roleTerms. Something should
+be treated as a subject if it is represented within a photograph and treated as a name with a roleTerm if the indvidual
+listed is associated with the creation or provenance of the item depicted. As the names values are currently represented
+in the metadata, these two distinct categories are mixed. The only place where the two can overlap is if the name
+is given a role of `"Depicted" - <https://id.loc.gov/vocabulary/relators/dpc>`_.
+
+Justification
+^^^^^^^^^^^^^
+
+As the names are currently represented, they share incorrect information.
+
+Xpath
+^^^^^
+
+    :code:`mods/subject/name[role]`
+
+Decision
+^^^^^^^^
+
+`Example record - arrsimple:344 <https://digital.lib.utk.edu/collections/islandora/object/arrsimple%3A344/datastream/MODS/view>`_
+This record is particularly problematic because it both suggests that Aunt Lydia Whaley is the photographer and that the
+photographer is unknown. This suggests that she cannot have the role of photographer. She may be depicted within the
+photograph, but this is also unconfirmed. While more vague, the safest way to deal with these records is to drop the
+roleTerm and treat the names as general subjects. For this record, the inclusion of this subject could mean that
+Aunt Lydia is the woman depicted, but it could also reinforce the title (Aunt Lydia's cave by the creek). It is
+ambiguous, but doesn't state anything that is clearly untrue.
+
+.. code-block:: xml
+
+    <subject authority="local">
+        <name>
+            <namePart>Whaley, Aunt Lydia.</namePart>
+        <role>
+            <roleTerm authority="marcrelator" valueURI="http://id.loc.gov/vocabulary/relators/pht">Photographer</roleTerm>
+        </role>
+        </name>
+    </subject>
+    <name>
+        <namePart>Unknown</namePart>
+        <role>
+            <roleTerm authority="marcrelator" valueURI="http://id.loc.gov/vocabulary/relators/pht">Photographer</roleTerm>
+        </role>
+    </name>
+    <title>Aunt Lydia's cave by the creek, Francisa.</title>
+
+.. code-block:: turtle
+
+    @prefix dcterms: <http://purl.org/dc/terms/> .
+
+    <https://example.org/objects/1> dcterms:subject "Whaley, Aunt Lydia"
```

Comment:
I know that you know this, but we're missing a `.` here at the end of 2416.

Reply:
Thanks @CanOfBees for noting it though - yesterday got away from me.

Reply:
:+1: